week1/week1-assigment/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "node.js example \u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[E\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C HTTP server",
```

Comment:
I don't mind you committing your package.json file, but be careful about how your 'description' seems to include a bunch of binary characters or something.

Reply:
okeeez